internal/provider/waypoint/data_source_waypoint_application.go

```diff
@@ -80,6 +80,26 @@ func (d *DataSourceApplication) Schema(ctx context.Context, req datasource.Schem
 				Computed:    true,
 				Description: "Internal Namespace ID.",
 			},
+			"app_input_vars": schema.SetNestedAttribute{
```

Comment:
Should we name them simply `input_variables`? The `app` prefix seems to stutter a bit (`application.app_input..`) and in the Template resource we use the full word `variables` and not just `vars`